hotTakeData.json

```diff
@@ -100,7 +110,9 @@
     "LESS",
     "Angular",
     "React",
-    "Vue"
+    "Vue",
+    "Malbolge",
+    "Zaskroniec"
```

Comment:
is this a language? google is giving me results for snakes

Reply:
https://github.com/kspalaiologos/Zaskroniec